test-sol/utils/Strategies.sol

```diff
@@ -349,4 +349,179 @@ contract Strategies {
 
         return boostStrategy.createStrategy();
     }
-}
\ No newline at end of file
+
+    function createAaveV3Repay() internal returns (uint256) {
+        StrategyBuilder repayStrategy = new StrategyBuilder("AaveV3Repay", true);
+
+        repayStrategy.addSubMapping("&targetRatio");
+        repayStrategy.addSubMapping("&checkRepayState");
+        repayStrategy.addSubMapping("&useDefaultMarket");
+        repayStrategy.addSubMapping("&useOnBehalf");
+
+        repayStrategy.addTrigger("AaveV3RatioTrigger");
+
+        string[] memory withdrawParams = new string[](5);
+        withdrawParams[1] = "&useDefaultMarket";
+        withdrawParams[3] = "&proxy";
+        repayStrategy.addAction("AaveV3Withdraw", withdrawParams);
+
+        string[] memory sellParams = new string[](5);
+        sellParams[0] = "&baseToken";
```

Comment:
I assume this is a leftover from the CompoundV3 tests, as there is no concept of a "`baseToken`" in AaveV3. The reason this works is that, during `repayStrategy.createStrategy()`, baseToken isn’t found in subMapping, and a default zero value is returned. Later, in DFSSell, this will be characterized as `NO_MAPPING`, so you need to explicitly pass a value when calling this strategy, as you currently do. Therefore, we can remove this. The same applies to other strategies.